MonoGame.Framework/Input/JoystickState.cs

```diff
@@ -19,7 +21,7 @@ public struct JoystickState
         /// Gets the joystick axis values.
         /// </summary>
         /// <value>An array list of floats that indicate axis values.</value>
```

Comment:
Can you look over the docs here a little closer... seems like it still references the axis as float.